cadc-postgresql-dev/README.md

```diff
@@ -0,0 +1,48 @@
+# base cadc-dev-postgresql image
+
+## Expected deployment
+This postgresql instance is designed for development support and has a very low level of
+security.
+
+## databases
+On startup, the following databases are created:
+
+cadctest : intended for database library integration tests
+content  : intended for web service back end integration tests
+
+## users
+The following user accounts are created (name : password):
+
+caom2 : pw-caom2
+invadm : pw-invadm
+
+tapadm : pw-tapadm
+tapuser : pw-tapuser
+
+These users are available in all databases.
+
+## schemas
+The following schemas are created (name : acccount with full permissions):
+
+caom2 : caom2
+inventory : invadm
+
+tap_schema : tapadm
+uws : tapadm
+
+These schemas are available in all databases. The first two (caom2 and inventory) are 
+for specific content; it would be feasible to have a single "content" schema and run 
+separate servers instead... TBD.
+
+## building it
+docker build -t cadc-postgresql-dev -f Dockerfile .
+
+## checking it
+docker run -it cadc-postgresql-dev:latest /bin/bash
```

Comment:
Does it come with `psql` installed? I assume that this would be the main tool for checking. Or can it be done from the client through an exposed port?

Reply:
It does have psql inside so you could do "docker exec .... psql" but I usually use psql on the host via the exposed port.